lib/diagnostic.js

```diff
@@ -8,39 +8,60 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-const min = function min() {
-
+const min = function min(prev, curr) {
+  if (curr < prev) {
+    return curr;
+  }
 };
 
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-const sum = function sum() {
-
-};
+const sum = function sum(prev, curr) {
+    return prev + curr;
+  };
 
 // Question 3
 // Write a function that returns true
 // if invoking the callback for any element
 // in the array returns a truthy value
 const any = function any(array, callback) {
-
-};
+  let result = false;
+     for (let i = 0; i < array.length; i++) {
+       if (!callback(array[i])) {
+         result = true;
+       }
+     }
+     return result;
+   };
 
```

Comment:
Extremely close here - you want to return `true` if the callback returns true, thus you'd want to check for ` if (callback(array[i])) {`